pkg/users/account-create-dialog.js

```diff
@@ -163,6 +192,11 @@ function suggest_username(realname) {
 }
 
 export function account_create_dialog(accounts) {
+    const uids = accounts.filter(a => a.name !== 'nobody').map(account => account.uid);
```

Comment:
This is an odd special case. There are lots of existing users which the cockpit user shouldn't touch, like all system users (< UID_MIN; same for groups, of course). This should already be taken care of by get_next_available_id(), no? If not, please move it there, this really shouldn't be in the caller of that function.

Reply:
This is also still current.

Reply:
Ah nevermind, this was at a different place. I'll add a new comment.